livekit-agents/livekit/agents/worker.py

```diff
@@ -781,3 +800,54 @@ async def _update_job_status(self, proc: ipc.job_executor.JobExecutor) -> None:
         )
         msg = agent.WorkerMessage(update_job=update)
         await self._queue_msg(msg)
+
+    async def _monitor_memory(self) -> None:
```

Comment:
I think we should move that to the PorcJobExecutor class. Otherwise looks good!

Reply:
Moved to the ProcJobExecutor